docs/HOW_TO_TEST.md

```diff
@@ -53,12 +56,14 @@ For more information about access key setup, see [this serverless guide](https:/
 Make sure your local environment meets the [requirements](https://github.com/ec-europa/eubfr-data-lake#requirements)
 
 Make sure your environment is ready
-  - `yarn`
+
+* `yarn`
 
 Deploy on root
-  - go to root of the project
-  - `yarn deploy`
-  - `yarn deploy-demo`
+
+* go to root of the project
+* `yarn deploy`
```

Comment:
Nothing about `yarn deploy-services`?

Reply:
Good point! For this, I just made a reference to the main README, which is the best source to explain this part.